net/instaweb/rewriter/image_rewrite_filter.cc

```diff
@@ -2136,13 +2136,18 @@ void ImageRewriteFilter::StartElementImpl(HtmlElement* element) {
     }
 
     BeginRewriteImageUrl(element, attributes[i].url);
-  }
-
-  if (element->keyword() == HtmlName::kImg) {
+   
+  
     HtmlElement::Attribute* srcset = element->FindAttribute(HtmlName::kSrcset);
+	  HtmlElement::Attribute* datasrcset = element->FindAttribute(HtmlName::kDataSrcset);
```

Comment:
nit: Can you fix the indentation here? Sorry about dragging on about that, but we're pretty strict about style; consistency helps everyone a lot.

Reply:
I will wait until let`s decide what to use to simplify the srcset and data-srcset if statement.

Reply:
chnaged and added an alternative to the double if